src/types/media.ts

```diff
@@ -45,7 +45,7 @@ export type BitmapImage = {
     mediaUrl?: string;
 };
 
-export type XpImage = VectorImage | BitmapImage;
+export type XpImageProps = VectorImage | BitmapImage;
```

Comment:
Når jeg leser *Props, så tenker jeg et objekt med diverse properties. Burde denne hete XpImageType istedet, bare for å være tydelig i navnekonvensjoner?

Reply:
Konvensjonen vi (løst) følger for slike type-navn er <komponentnavn>Props, dette er jo da _typen_ til et objekt med properties. Om vi bør være enda mer tydelige på at dette er en type kan diskuteres.